docs/pages/guides/schema-validation.mdx

```diff
@@ -0,0 +1,124 @@
+---
+meta:
+  title: "Schema Validation"
+  parentTitle: "Guides"
+  description: "Learn how to use Schema Validation to ensure data integrity in your application "
+---
+
+## Why Schema Validation™?
+Schema validation is an important feature for future-proofing your application and ensuring that implementing new features or data structures into your app does not break the integration you have established with Liveblocks.  Our Storage block is flexible in the sense that you can use it to store different Type structures dependent upon your unique needs.  What happens though, if you alter your application by updating some or all of the properties of an object in Storage and you have existing data?
```

Comment:
```suggestion
Schema validation is an important feature for future-proofing your application and ensuring that implementing new features or data structures into your app does not break the integration you have established with Liveblocks. Our conflict-free data types are flexible in the sense that you can use them to store different Type structures dependent upon your unique needs. What happens though, if you alter your application by updating some or all of the properties of an object in storage and you have existing data?
```

We shouldn't say "presence block". or "storage block", now we have four main features for collaborative documents, as you can see in the tree map here: https://preview.liveblocks.io/

- Presence
- Broadcast
- Conflict-free data types
- Comments (soon)

And why is Type with an uppercase first letter?